api/src/main/java/org/openmrs/module/referencemetadata/OrderFrequencyMetaData.java

```diff
@@ -0,0 +1,198 @@
+		install(orderFrequency(_Concept.EVERY_30_MINS, _OrderFrequency.EVERY_30_MINS, 48.0));
+		install(orderFrequency(_Concept.EVERY_HOUR, _OrderFrequency.EVERY_HOUR, 24.0));
+		install(orderFrequency(_Concept.EVERY_TWO_HOURS, _OrderFrequency.EVERY_TWO_HOURS, 12.0));
+		install(orderFrequency(_Concept.EVERY_THREE_HOURS, _OrderFrequency.EVERY_THREE_HOURS, 8.0));
+		install(orderFrequency(_Concept.EVERY_FOUR_HOURS, _OrderFrequency.EVERY_FOUR_HOURS, 6.0));
+		install(orderFrequency(_Concept.EVERY_FIVE_HOURS, _OrderFrequency.EVERY_FIVE_HOURS, 4.8));
+		install(orderFrequency(_Concept.EVERY_SIX_HOURS, _OrderFrequency.EVERY_SIX_HOURS, 4.0));
+		install(orderFrequency(_Concept.EVERY_EIGHT_HOURS, _OrderFrequency.EVERY_EIGHT_HOURS, 3.0));
+		install(orderFrequency(_Concept.EVERY_TWELVE_HOURS, _OrderFrequency.EVERY_TWELVE_HOURS, 2.0));
+		install(orderFrequency(_Concept.TWICE_DAILY, _OrderFrequency.TWICE_DAILY, 2.0));
+		install(orderFrequency(_Concept.TWICE_DAILY_BEFORE_MEALS, _OrderFrequency.TWICE_DAILY_BEFORE_MEALS, 2.0));
+		install(orderFrequency(_Concept.TWICE_DAILY_AFTER_MEALS, _OrderFrequency.TWICE_DAILY_AFTER_MEALS, 2.0));
+		install(orderFrequency(_Concept.TWICE_DAILY_WITH_MEALS, _OrderFrequency.TWICE_DAILY_WITH_MEALS, 2.0));
+		install(orderFrequency(_Concept.EVERY_24_HOURS, _OrderFrequency.EVERY_24_HOURS, 1.0));
+		install(orderFrequency(_Concept.ONCE_DAILY, _OrderFrequency.ONCE_DAILY, 1.0));
+		install(orderFrequency(_Concept.ONCE_DAILY_BEDTIME, _OrderFrequency.ONCE_DAILY_BEDTIME, 1.0));
+		install(orderFrequency(_Concept.ONCE_DAILY_EVENING, _OrderFrequency.ONCE_DAILY_EVENING, 1.0));
+		install(orderFrequency(_Concept.ONCE_DAILY_MORNING, _OrderFrequency.ONCE_DAILY_MORNING, 1.0));
+		install(orderFrequency(_Concept.THRICE_DAILY, _OrderFrequency.THRICE_DAILY, 3.0));
+		install(orderFrequency(_Concept.THRICE_DAILY_AFTER_MEALS, _OrderFrequency.THRICE_DAILY_AFTER_MEALS, 3.0));
+		install(orderFrequency(_Concept.THRICE_DAILY_BEFORE_MEALS, _OrderFrequency.THRICE_DAILY_BEFORE_MEALS, 3.0));
+		install(orderFrequency(_Concept.THRICE_DAILY_WITH_MEALS, _OrderFrequency.THRICE_DAILY_WITH_MEALS, 3.0));
+		install(orderFrequency(_Concept.FOUR_TIMES__DAILY_WITH_MEALS, _OrderFrequency.FOUR_TIMES__DAILY_WITH_MEALS, 4.0));
+		install(orderFrequency(_Concept.FOUR_TIMES__AFTER_WITH_MEALS_BEDTIME,
+				_OrderFrequency.FOUR_TIMES__AFTER_WITH_MEALS_BEDTIME, 4.0));
+		install(orderFrequency(_Concept.FOUR_TIMES__BEFORE_WITH_MEALS_BEDTIME,
+				_OrderFrequency.FOUR_TIMES__BEFORE_WITH_MEALS_BEDTIME, 4.0));
+		install(orderFrequency(_Concept.EVERY_48_HOURS, _OrderFrequency.EVERY_48_HOURS, 0.5));
+		install(orderFrequency(_Concept.EVERY_36_HOURS, _OrderFrequency.EVERY_36_HOURS, (1 / 36) * 24.0));
+		install(orderFrequency(_Concept.EVERY_72_HOURS, _OrderFrequency.EVERY_72_HOURS, (1 / 72) * 24.0));
+		install(orderFrequency(_Concept.MONDAY_WEDNESDAY_FRIDAY, _OrderFrequency.MONDAY_WEDNESDAY_FRIDAY, (3 / 168) * 24.0));
+	}
+
+	public static class _OrderFrequency {
```

Comment:
Did you get this class naming convention from here? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions